pandas_datareader/av/__init__.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from pandas_datareader.base import _BaseReader
+from pandas_datareader._utils import RemoteDataError
+
+import pandas as pd
+
+AV_BASE_URL = 'https://www.alphavantage.co/query'
+
+
+class AlphaVantage(_BaseReader):
+    """
+    Base class for all AlphaVantage queries
```

Comment:
I think there should be docstring here.  Maybe we should use a doc string inheriter to simplify this?

Reply:
Nevermind - this is a base class.